velox/core/PlanNode.cpp

```diff
@@ -1288,6 +1290,7 @@ PlanNodePtr WindowNode::create(const folly::dynamic& obj, void* context) {
       sortingOrders,
       windowNames,
       functions,
+      false, /*isStreamingWindow*/
```

Comment:
This is not right. You need to update serialization logic to include this field and restore it here. Please, make sure to add tests.

Also, make sure to include this flag in the output of toString() + add tests.

Reply:
Done.